Formula/node@18.rb

```diff
@@ -0,0 +1,110 @@
+class NodeAT18 < Formula
+  desc "Platform built on V8 to build network applications"
+  homepage "https://nodejs.org/"
+  url "https://nodejs.org/dist/v18.11.0/node-v18.11.0.tar.xz"
+  sha256 "8b9643dc6fce79c1e99379db0ce64e43601e2e2d7389015fe8985cc4ccd0ce17"
+  license "MIT"
+
+  livecheck do
+    url "https://nodejs.org/dist/"
+    regex(%r{href=["']?v?(18(?:\.\d+)+)/?["' >]}i)
+  end
+
+  keg_only :versioned_formula
+
+  # https://nodejs.org/en/about/releases/
+  # disable! date: "2025-04-30", because: :unsupported
+  deprecate! date: "2023-10-18", because: :unsupported
+
+  depends_on "pkg-config" => :build
+  depends_on "python@3.10" => :build
+  depends_on "brotli"
+  depends_on "c-ares"
+  depends_on "icu4c"
+  depends_on "libnghttp2"
+  depends_on "libuv"
+  depends_on "openssl@1.1"
+
+  uses_from_macos "python", since: :catalina
+  uses_from_macos "zlib"
+
+  on_macos do
+    depends_on "llvm" => [:build, :test] if DevelopmentTools.clang_build_version <= 1100
+  end
+
+  fails_with :clang do
+    build 1100
+    cause <<~EOS
+      error: calling a private constructor of class 'v8::internal::(anonymous namespace)::RegExpParserImpl<uint8_t>'
+    EOS
+  end
+
+  fails_with gcc: "5"
+
+  def install
+    ENV.llvm_clang if OS.mac? && (DevelopmentTools.clang_build_version <= 1100)
+
+    # make sure subprocesses spawned by make are using our Python 3
+    ENV["PYTHON"] = which("python3.10")
+
+    # Never install the bundled "npm", always prefer our
+    # installation from tarball for better packaging control.
```

Comment:
That comment does not apply here because for the versioned formulae we do use the bundled NPM.

```suggestion
```